### DISCUSSION TITLE
Detail about fixed issue in 3.4.x

### DISCUSSION THREAD

**Discussion by atsushi-tsuda** [containers/podman#12554] [Q&A] [answered]
I'm working on software development with containers on podman 3.3.1, which users can be installed via yum command.

I want to know the detail (Cause of the issues / condition / workaround) about the following fixed bugs in 3.4.x to check whether I might meet them with my usage of podman or not. And if yes, I want to know how to avoid them.

I searched issues and PRs with some keywords, but I could not find.

Can anyone help me to give the detail?

- v3.4.2: Fixed a bug where built-in volumes specified by images would not be created correctly under some circumstances.
- v3.4.0: Fixed a bug where the podman info command could segfault when accessing cgroup information.

**Comment by mheon** [comment, marked as answer]
The image issues was https://github.com/containers/podman/pull/12127

The info issue was https://github.com/containers/podman/pull/11463

**Reply by atsushi-tsuda** [reply]
Thanks a lot! Now I can investigate them.
